---
title: "Accurate inference of genetic ancestry from cancer-derived sequencing"
author: Pascal Belleau, Astrid Deschênes and Alexander Krasnitz
output:
  rmarkdown::html_document:
    highlight: pygments
    toc: true
    fig_width: 5
    number_sections: true
pkgdown:
  asis: true
bibliography: aics.bibtex
vignette: >
  %\VignetteIndexEntry{Accurate inference of genetic ancestry from cancer-derived sequencing}
  %\VignettePackage{aics}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r style, echo = FALSE, results = 'asis', warning=FALSE, message=FALSE}
BiocStyle::markdown()

suppressPackageStartupMessages({
  library(knitr)
  library(aics)
})

set.seed(121444)
```

# Licensing 

TODO

<br>
<br>

# Citing

If you use this package for a publication, we would ask you 
to cite the following:

TODO

# Installation

To install the latest version accessible on the 
[aics Github Website](https://github.com/adeschen/aics "aics Github Site"), 
the `r CRANpkg("devtools")` package is required.

```{r installDemo01, eval=FALSE}
## Load required package
library(devtools)

## Install the latest version of aics
devtools::install_github('adeschen/aics')
```

It is also possible to install an official release. The list of available 
releases is posted on the [aics Release Website](https://github.com/adeschen/aics/releases "aics Release Site").


```{r installDemo02, eval=FALSE}
## Load required package
library(devtools)

## Install the version v0.1.4 of aics
## using 'ref' parameter
devtools::install_github('adeschen/aics', ref = "v0.1.4")
```

<br>
<br>

# Introduction


Epidemiological evidence point to race and/or ethnicity as 
important determinants of incidence and outcome in multiple 
types of cancer [@Cronin2018; @Siegel2020].
Ancestry annotation of cancer data often draws from 
patient’s self-identified race and/or ethnicity (SIRE). 
Yet, SIRE information is frequently missing or incomplete [@Nugent2019]. 
Moreover, SIRE information is not always consistent with genetic ancestry.

Accurate ancestral characterization can also be obtained by genotyping a 
patient's DNA from a cancer-free tissue. 
Multiple methods have been implemented to infer ancestry from germline DNA 
sequence [@Price2006; @Pritchard2000; @Alexander2009]. However, genotyping of 
DNA from matched normal specimens is not part of standard clinical practice
and is not performed routinely outside academic clinical centers. 
In sum, matched germline DNA sequence is often missing for cancer-derived 
molecular data. 
In such cases, having the possibility to infer ancestry 
from tumor-derived data would be beneficial. 
However, accurate inference of ancestry is challenging for tumor-derived 
sequence.
Tumor genome are frequently filled with somatic alterations sush as 
single nucleotide variants (SNV), copy number variants, translocations and microsatellite instabilities. 

The **aics** package implements an inference procedure that has been
specifically developed to accurately infer genetic ancestry from 
cancer-derived sequencing. The covered cancer-derived sequencing are, more 
specifically, tumor exomes, targeted gene panels and RNA sequences.


The **aics** package also implements simulation technique that generates
synthetic cancer-derived data. Synthetic cancer-derived data can be used to 
assess the performance of cancer related inference tools.

<br>
<br>


# Session info

Here is the output of `sessionInfo()` on the system on which this document was 
compiled:

```{r sessionInfo, echo=FALSE}
sessionInfo()
```

<br>
<br>


# References
